### PR TITLE
passes ACL argument, makes stdout more colorful

### PR DESCRIPTION
**Pull request by cerealcommas**
- uses chalk to print more verbose and colorful success message

- allows user to specify ACL policy (defaults to private)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -57,6 +58,10 @@ function setup(options) {
     return Promise.reject('Must specify bucket');
   }
 
+  if (!config.acl) {
+    config.acl = 'private';
```
Comment (JeremyPlease) [inline index.js]: Since the previous behavior was to use "public-read", let's set the default to "public-read".

File: README.md [hunk 2]
```diff
@@ -71,11 +74,12 @@ specified in ~/.aws/credentials. This can be specified instead of `accessKeyId`
     --region <region>                           The S3 region. Defaults to us-east-1
     --public-root <publicRoot>                  The path of the folder to deploy
     --bucket <bucket>                           The S3 bucket name
+    --acl <acl>                                 The ACL policy. Defaults to private
```
Comment (JeremyPlease) [inline README.md]: Let's default to public-read to match previous behavior.

**Review by JeremyPlease — CHANGES_REQUESTED** [review]
Thanks for the PR! And thanks for fixing some typos and line-ending issues.

Only thing to change is to default the acl to "public-read" to match the previous behavior.

**Comment by cerealcommas** [comment]
Done and pushed!

**Review by JeremyPlease — APPROVED** [review]

**Comment by JeremyPlease** [comment]
Thanks @cerealcommas! Version v0.2.0 published to NPM

**Pull request merged**